### ISSUE TITLE
Add EasyList Polish ?

### ISSUE THREAD

**Issue by AroKol78** [uBlockOrigin/uBlock-issues#1336]
Could you please add "EasyList Polish" to the Regions, languages.
they complement each other
Regions, languages (2/34)
POL: Oficjalne Polskie Filtry do AdBlocka, uBlocka Origin i AdGuarda​​​​​ użytych 7 100 z 7 233​
POL: Oficjalne polskie filtry przeciwko alertom o Adblocku​​​​​ użytych 1 146 z 1 158

EasyList Polish​​​​​ użytych 3 677 z 3 835

EasyList Polish :
https://easylist-downloads.adblockplus.org/easylistpolish.txt
https://github.com/easylistpolish/easylistpolish

I'm using uBlock-for-firefox-legacy

**Comment by uBlock-user** [comment]
> Opening issues for adding new filter lists is now disallowed and such issues will be declined and closed.

**Comment by gorhill** [comment]
uBO already includes POL lists which make use of uBO's extended syntax, there is no point adding a list which limits itself to only ABP syntax.

**Comment by easylistpolish** [comment]
Hey,
First at all, thank you people for suggesting our list to be added to uBO, I'm happy to hear that you find it useful.:)

That is true that we are creating filters following ABP syntax. Anyway the majority of the filters should work also in uBO. Our combined files have >2600 filters in which 9 are typical for ABP (#?# filters).

We also do not treat "Official Polish Filters for AdBlock, uBlock Origin & AdGuard" as competitors. We are working for the same purpose with a bit of a different approach so fighting with each other doesn't make any sense IMO. We just work in parallel and as I can read they have better results on some websites and we have better results on other ones (this is just a guess but I assume that we are more focused to cover more websites, even small ones, what drops our quality on more popular ones, so they have much better quality on the most popular ones - sorry if I'm wrong, as I said this is just my assumption).

About not used filters. As I guess, this is simply caused by duplicates, which are natural if we are sometimes writing filters for the same ads on the same websites. If we see, for example, some class which contains only ads on some website, we will write example.com##.class and most of the people who write filters will do the same. Depending on which list you add as first, filters from different lists are taken as used. Here I added first EL Polish and then Oficjalne Polskie Filtry, and then opposite and that's what uBO showed me:
![image](https://user-images.githubusercontent.com/26573731/99524148-4d163180-2998-11eb-9270-5753c2e7ae8f.png)

If there are users who see it useful, maybe it would be worth considering adding it to the list of optional filters, especially if ABP syntax filters are <0.5% of all filters on that list (AFAIK about which filters are not compatible with uBO).
Of course this is fully your decision, should it be added or not and I'm not going to anyhow push for it.

Best

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/301
